### PR TITLE
Create DF Features Highlight component

### LINKED ISSUE #542: Feature Highlighting & Explanation
Rewards Page

Make a 3 columns explainers for these features:
- Ranked Rewards
- Publishers 2x
- NO IL ("No Imperative Loss")

the goal is allowing users to have more context of the app usage.

### PR DESCRIPTION
**Pull request by KatunaNorbert**
Fixes #542.

### CODE REVIEW

**Comment by KatunaNorbert** [comment]
Maybe we could display other features like: Earn by holding, Earn by allocating, Weekly rewards.
We could also have a carousel that contains multiple features.

**Comment by KatunaNorbert** [comment]
Another idea is to make cards clickable and link them to docs section for that specific feature

**Comment by KatunaNorbert** [comment]
We might also consider including this components inside the Round Summary/Countdown component. We can earn some extra space and give a title/context to this section

**Comment by idiom-bytes** [comment]
I only noticed this now...
These are looking great. Fantastic work!

I added some new comments to the ticket, but I'll turn it into a new ticket (FTUE-oriented design).
Let's launch this as is, then iterate!

File: src/utils/metadata/rewards/features.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "description": "Reward amounts are distributed exponentially to the fist 50 assets with consume volume",
+        "image": "/images/features/podium.png"
+    },
+    {
+        "title": "Publisher 2x",
+        "description": "Publisher gets 2x allocation power when allocating on his own assets",
+        "image": "/images/features/exchange.png"
+    },
+    {
+        "title": "No IL",
```
[the remark this answers is not in the document]
Comment (idiom-bytes) [reply src/utils/metadata/rewards/features.json:13]: /Agree

File: src/utils/metadata/rewards/features.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "title": "Ranked Rewards",
+        "description": "Reward amounts are distributed exponentially to the fist 50 assets with consume volume",
+        "image": "/images/features/podium.png"
+    },
+    {
+        "title": "Publisher 2x",
```
[the remark this answers is not in the document]
Comment (idiom-bytes) [reply src/utils/metadata/rewards/features.json:8]: Screenshot from 2023-04-06 20-35-55

In the app, we're calling it "2x Publishers".
In the docs, we're calling it ["2x Stake"](https://docs.oceanprotocol.com/veocean-data-farming/df-intro#publisher-rewards-2x-stake)

(1) - For now, rename it as "2x Publishers"
(2) - I'll create a separate ticket to rename all of this to "2x Stake" so it's 1:1 with the docs

Whenever possible, always use the same terminology as the docs, etc... my bad, we'll keep improving.

[the remark this answers is not in the document]
Comment (KatunaNorbert) [reply src/utils/metadata/rewards/features.json:8]: This looks like a small change. I think there is no need of a separate issue for this, let me change it here so it is going to link better to docs.

**Comment by KatunaNorbert** [comment]
I'm gonna do the following changes before getting it reddy for review:
- change 2x allocations to 2x stake
- link cards to docs when clicking on them

**Comment by EnzoVezzaro** [comment]
I think this is good. Thank you @KatunaNorbert 🙏🏾 I was planing to do it when back on it, but thanks for taking it on you. 

All seems good, I'd just suggest to remove the zoom in/our effect. I'd just keep the link icon transition next to the title instead of the right corner. Like so:

Screenshot 2023-04-10 at 09 00 38

If you can, also apply a fade in/out when going from top to x on the animation of the link when the card is hovered.

**Comment by jamiehewitt15** [comment]
I think it would be worth adjusting the padding slightly so that "no impermanent loss" fits on one line 

Currently:
image

On one line:

image

File: src/utils/metadata/rewards/features.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "title": "Ranked Rewards",
+        "description": "Reward amounts are distributed exponentially to the fist 50 assets with consume volume",
```
Comment (jamiehewitt15) [inline src/utils/metadata/rewards/features.json:4]:
```suggestion
        "description": "Reward amounts are distributed exponentially to the first 50 assets with consume volume",
```

File: src/utils/metadata/rewards/features.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    },
+    {
+        "title": "Publisher 2X Stake",
+        "description": "Publisher gets 2x allocation power when allocating on his own assets",
+        "image": "/images/features/exchange.png",
+        "link": "https://docs.oceanprotocol.com/veocean-data-farming/df-intro#publisher-rewards-2x-stake"
+    },
+    {
+        "title": "No Impermanent Loss",
+        "description": "No OCEAN tokens lost when locking in echange for veOCEAN",
```
Comment (jamiehewitt15) [inline src/utils/metadata/rewards/features.json:16]:
```suggestion
        "description": "No OCEAN tokens lost when locking in exchange for veOCEAN",
```

File: src/utils/metadata/rewards/features.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "title": "Ranked Rewards",
+        "description": "Reward amounts are distributed exponentially to the fist 50 assets with consume volume",
+        "image": "/images/features/podium.png",
+        "link": "https://docs.oceanprotocol.com/veocean-data-farming/df-intro#ranked-rewards"
+    },
+    {
+        "title": "Publisher 2X Stake",
+        "description": "Publisher gets 2x allocation power when allocating on his own assets",
```
Comment (jamiehewitt15) [inline src/utils/metadata/rewards/features.json:10]:
```suggestion
        "description": "Publishers get 2x allocation power when allocating on their own assets",
```

**Review by jamiehewitt15 — APPROVED** [review]
Looks good to me

File: src/utils/metadata/rewards/features.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (jamiehewitt15) [reply src/utils/metadata/rewards/features.json]: This is done

**Pull request merged**